cargo-paradedb/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "cargo-paradedb"
+version = "0.1.0"
```

Comment:
Should this be the same version as everything else?

Reply:
Yes, that's a good idea. I've changed it.